reference/fleet/remote-elasticsearch-output.md

```diff
@@ -57,4 +59,37 @@ After the output is created, you can update an {{agent}} policy to use the new r
 
 The remote {{es}} cluster is now configured.
 
-As a final step before using the remote {{es}} output, you need to make sure that for any integrations that have been [added to your {{agent}} policy](/reference/fleet/add-integration-to-policy.md), the integration assets have been installed on the remote {{es}} cluster. Refer to [Install and uninstall {{agent}} integration assets](/reference/fleet/install-uninstall-integration-assets.md) for the steps.
+If you have chosen not to automatically synchronize integrations, you need to make sure that for any integrations that have been [added to your {{agent}} policy](/reference/fleet/add-integration-to-policy.md), the integration assets have been installed on the remote {{es}} cluster. Refer to [Install and uninstall {{agent}} integration assets](/reference/fleet/install-uninstall-integration-assets.md) for the steps.
+
+## Automatic integrations synchronization
+
+```{applies_to}
+stack: ga 9.1
+```
+
+When enabled, this feature keeps integrations synced between your main {{es}} cluster and remote {{es}} clusters.
+
+### Requirements
+
+This feature requires setting up [{{ccr}}](/deploy-manage/tools/cross-cluster-replication.md), which is available to Platinum and Enterprise [subscriptions](https://www.elastic.co/subscriptions). Remote clusters must be running the same version of {{es}} as the main cluster or a newer version that is compatible with {{ccr}}.
+
+Remote clusters require access to the [{{package-registry}}](/reference/fleet/index.md#package-registry-intro) to install integrations.
+
+### Configuration
+
+1. Configure {{ccr}} on the remote cluster.
+
+    1. In the remote cluster, open the {{kib}} menu and go to **Stack Management > Remote Clusters**.
+    2. Refer to [Remote clusters](https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters.html) to add your main cluster (where the remote {{es}} output is configured) as a remote cluster.
+    3. Go to **Stack Management > Cross-Cluster Replication**.
+    4. Create a follower index named `fleet-synced-integrations-ccr-<output name>` that replicates the `fleet-synced-integrations` leader index on the main cluster.
```

Comment:
We could add one more step to `Resume replication` after adding the follower index.